debug.log

```diff
@@ -15,3 +15,4 @@
 [1130/214333.166:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
```

Comment:
See comment in https://github.com/green-fox-academy/IceAge-Reddit-Frontend/pull/7/files